packages/nuxt/src/app/composables/loading-indicator.ts

```diff
@@ -50,7 +61,12 @@ function createLoadingIndicator (opts: Partial<LoadingIndicatorOpts> = {}) {
 
   let _throttle: any = null
 
-  const start = () => set(0)
+  const start = (force: boolean | null = true) => {
```

Comment:
What about updating the signature so we call this with `start({ force: true})` (and also below with `finish`)? This makes it more explicit and allows passing more options in future.

Reply:
Sorry, to clarify I don't mean passing it as an option to `useLoadingIndicator` but to the `start` and `finish` functions.

Reply:
Hi Daniel!

I thought about it as well but the better would be passing force as parameter to loading indicator instead.

Let's say:
1. LoadingIndicator starts
2. We try to start the new one with `start({force: true})`.
3. LoadingIndicators starts again.
4. `_hide` function is being called and LoadingIndicator is being hidden because it was still running because of setTimeout...

Let me know what do you think!

Thanks!

Reply:
The fewer options that `useLoadingIndicator` accepts, the better, because they will influence the global singleton.

I think we should pass the override to start/finish and calling `_hide` or `_reset` should instead clear any timeout that is still running. (We probably should have implemented it that way in the first place.)

Reply:
Hi @danielroe,

I've applied your suggestions, let me know wdyt.

Thanks!

Reply:
Thank you!